site/src/pages/component-health.tsx

```diff
@@ -30,94 +58,124 @@ export default ({ data, location }) => {
-                    <Paragraph variant="body" tag="span">
-                      {mdx.node.frontmatter.title}
-                    </Paragraph>
+              </tbody>
+              <tfoot>
+                <tr>
+                  <td>
+                    <Heading tag="span" variant="heading-4">
+                      Total ({componentsFiltered.length})
+                    </Heading>
                   </td>
                   {healthAttributeMap.map(attribute => {
-                    if (!mdx.node.frontmatter.health) {
-                      return <td key={attribute.id}></td>
-                    }
-
+                    const count = totals[attribute.id]
+                    const percentage = (count / componentsFiltered.length) * 100
                     return (
-                      <td
-                        key={attribute.id}
-                        className={
-                          mdx.node.frontmatter.health[attribute.id]
-                            ? styles.positive
-                            : styles.negative
-                        }
-                      >
-                        {mdx.node.frontmatter.health[attribute.id] ? (
-                          <Icon
-                            icon={successIcon}
-                            title={attribute.positive}
-                            role="img"
-                          />
-                        ) : null}
+                      <td key={attribute.id}>
+                        <Heading tag="span" variant="heading-4">
+                          {count}
+                        </Heading>
+
+                        <Paragraph tag="span" variant="small">
+                          ({percentage.toFixed(0)}%)
+                        </Paragraph>
                       </td>
                     )
                   })}
                 </tr>
-              ))}
-            </tbody>
-            <tfoot>
-              <tr>
-                <td>
-                  <Heading tag="span" variant="heading-4">
-                    Total ({componentsFiltered.length})
-                  </Heading>
-                </td>
-                {healthAttributeMap.map(attribute => {
-                  const count = totals[attribute.id]
-                  const percentage = (count / componentsFiltered.length) * 100
-                  return (
-                    <td key={attribute.id}>
-                      <Heading tag="span" variant="heading-4">
-                        {count}
-                      </Heading>
+              </tfoot>
+            </table>
 
-                      <Paragraph tag="span" variant="small">
-                        ({percentage.toFixed(0)}%)
-                      </Paragraph>
-                    </td>
-                  )
-                })}
-              </tr>
-            </tfoot>
-          </table>
-        </Content>
-      </ContentOnly>
-    </Layout>
+            <Divider variant="content" />
+
+            <Box mb={2}> </Box>
```

Comment:
This looks a bit odd. Probably should surround something 🤷🏻

Reply:
Haha, I wondered if anyone would pick up on this. This is essentially the good old "pixel gif spacer" hack applied to our design system ;) I think I can actually wrap the `Divider` in it, but the visual outcome is the same.

Reply:
Yeah, I was more thinking of how we should expect the components to be used :P We shouldn't do things we don't expect our consumers not to do :)